packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -115,12 +115,11 @@ const NuovaDelega = () => {
 
   const xsValue = isMobile ? 12 : 4;
 
-  useEffect(
-    () => () => {
-      dispatch(resetNewDelegation());
-    },
-    []
-  );
+  useEffect(() => {
+    void dispatch(getAllEntities());
+    return () => { dispatch(resetNewDelegation()); };
```

Comment:
questo return a cosa serve? non basta dire void dispatch(resetNewDelegation());

Reply:
La return l'ho messa in quanto al componentDidMount bisogna resettare il form con l'azione resetNewDelegation